crates/uv/tests/it/pip_install.rs

```diff
@@ -486,6 +486,39 @@ fn install_requirements_txt() -> Result<()> {
     Ok(())
 }
 
+/// Warn (but don't fail) when unsupported flags are set in the `requirements.txt`.
+#[test]
+fn install_unsupported_flag() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let requirements_txt = context.temp_dir.child("requirements.txt");
+    requirements_txt.write_str(indoc! {r"
+        --pre
+        --prefer-binary :all:
+        iniconfig
+    "})?;
+
+    uv_snapshot!(context.pip_install()
```

Comment:
ooh neat, i hadn't noticed y'all using the inline snapshot magic yet

Reply:
It can always use some love if you're interested :)